All_home_works/hw2/task1.py

```diff
@@ -0,0 +1,112 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from typing import List
+
+CROP_PUNCTUATION = str.maketrans({char: "" for char in string.punctuation})
+
+
+class Counter:
+    def __init__(self):
+        self.counter = {}
+
+    def count(self, key):
+        if key in self.counter:
+            self.counter[key] += 1
+        else:
+            self.counter[key] = 0
+
+    def most_common(self):
```

Comment:
What is the use case for this method?

Reply:
it can be @property. Or I did not understand